src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixPlugin.groovy

```diff
@@ -81,12 +81,14 @@ class ScalafixPlugin implements Plugin<Project> {
             scalafixTask.compileOptions.set(project.provider({ sourceSet.compilerOptions }))
             scalafixTask.semanticdbConfigured = extension.semanticdb.autoConfigure.get()
 
-            if (extension.semanticdb.autoConfigure.get()) {
-                // configures the semanticdb compiler plugin during the execution phase, but before the
-                // compile task is executed. This prevents dependencies from being resolved too early
-                sourceSet.compileTask.doFirst {
+            sourceSet.compileTask.doFirst {
```

Comment:
should we consider replacing the `doFirst` action with a "configure task" as suggested [here](https://docs.gradle.org/current/userguide/common_caching_problems.html#custom_actions)?